spec.html

```diff
@@ -248,8 +312,10 @@ <h1>Intl.Locale.prototype.toString ()</h1>
         1. If _value_ is not *undefined*,
           1. Let _kvp_ be a String value produced by concatenating `"-"`, _key_, `"-"`, and ! ToString(_value_).
           1. Set _unicodeExt_ to a String value produced by concatenating _unicodeExt_ and _kvp_.
-      1. If _unicodeExt_ is not empty, then:
-        1. Let _result_ be a String value produced by concatenating _locale_, `"-u"`, and _unicodeExt_.
+      1. Set _unicodeExt_ to a String value produced by concatenating _unicodeExt_ and _loc_.[[OtherExtensions]].
```

Comment:
Is concatenating a string and  `string | undefined` generally allowed in the spec/assumed that engines will check that _loc_.[[OtherExtensions]] is not undefined first? A strict following of the spec could produce `"-u-co-phonebk" + undefined ->"-u-co-phonebkundefined"` here, no?

Reply:
Also, nit - should the name remain `unicodeExt` if it explicitly includes unicode + other extensions?

Reply:
When would `[[OtherExtensions]]` be undefined? I thought it'd always be a string. (My reading of spec formalisms is that such a cast wouldn't happen implicitly in spec-ese and simply be an editorial bug.)

Renamed _unicodeExt_ to _extension_.